mission 8/src/app.ts

```diff
@@ -0,0 +1,45 @@
+import express from "express";
+import cookieParser from "cookie-parser";
+import router from "./routers/index";
+import { PORT } from "./lib/constants";
+import passport from "./lib/passport/index";
+import cors from "cors";
+import { errorHandler } from "./middlewares/errorHandler";
+import http from "http";
+import { Server } from "socket.io";
+import { registerSocket } from "./socket";
+
+const app = express();
+
+app.use(express.urlencoded({ extended: true }));
+
+app.use(express.json());
+app.use(cookieParser());
+
+const corsOptions = {
+  origin: "http://localhost:3000",
+  optionSuccessStatus: 200,
```

Comment:
옵션명이 잘못되었어요. ([공식문서](https://expressjs.com/en/resources/middleware/cors.html#configuration-options))
```suggestion
  optionsSuccessStatus: 200,
```